.taskcluster.yml

```diff
@@ -159,7 +159,10 @@ tasks:
                       routes:
                         $flattenDeep:
                           - checks
-                          - $if: 'level == "3"'
+                          # We check for `!isPullRequest` instead of level 3 to support `staging-application-services`.
+                          # This is ok as ${project} uniquely identifies the repo within this trust domain, and each
+                          # project only has scopes to its own index namespace, so namespace collisions shouldn't happen.
+                          - $if: 'tasks_for != "github-pull-request"'
                             then:
                                 # TODO Bug 1601928: Make this scope fork-friendly once ${project} is better defined. This will enable
                                 # staging release promotion on forks.
```

Comment:
Maybe remove or update this TODO?  I think eventually we'd want to switch these to be under `{trust_domain}.v2.{alias}` (per trust-domain-scopes)?